src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Vector;
+
 public class Duke {
+    public static void takeCommand(String command,List<Task> tasks){
+        String[] subStrs = command.split(" ");
+
+        switch(subStrs[0]){
+        case "list":
+            printTasks(tasks);
+            break;
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            break;
+        case "done":
+            int taskNo = Integer.parseInt(subStrs[1]);
+            Task temp = tasks.get(taskNo-1);
```

Comment:
Naming a task object "temp" may be confusing to readers. May want to specify what exactly is this Task Object